buildpacks/README.md

```diff
@@ -0,0 +1,44 @@
+# Buildpack V3
+
+This build template builds source into a container image using [Cloud Native Buildpacks](https://buildpacks.io).
+
+The Cloud Native Buildpacks website describes v3 buildpacks as:
+
+> ... pluggable, modular tools that translate source code into container-ready artifacts
+> such as OCI images. They replace Dockerfiles in the app development lifecycle with a higher level
+> of abstraction. ...  Cloud Native Buildpacks embrace modern container standards, such as the OCI
+> image format. They take advantage of the latest capabilities of these standards, such as remote
+> image layer rebasing on Docker API v2 registries.
+
+**Note**: The current Cloud Foundry buildpacks are available in the [CF template](README-CF.md).
```

Comment:
This may seem obvious to you, but what about clearly stating that "the current CF buildpacks" are *not* v3 buildpacks, which we're dealing with here?

Reply:
We could explicitly call this out now. However, v3 compatible CF buildpacks are being developed and will be released in coming months.